packages/site/src/components/ComponentDoc.js

```diff
@@ -184,3 +183,7 @@ function renderExamples(
     </div>
   );
 }
+
+function renderPropDoc(propDoc: Object) {
+  return [<H3 key={0}>Props</H3>, <PropTable key={1} exampleProps={propDoc} />];
```

Comment:
Didn't realize we could return arrays of JSX elements. Will not miss the endless wrapping `div`s.